test/test_oneview_appliance_device_snmp_v1_trap_destinations.py

```diff
@@ -34,38 +34,21 @@
 PARAMS_FOR_PRESENT = dict(
```

Comment:
Do we really need to import mock library?

Reply:
removed